mlflow/pyfunc/model.py

```diff
@@ -183,6 +186,22 @@ def _save_model_with_class_artifacts_params(path, python_model, artifacts=None,
 def _load_pyfunc(model_path):
     pyfunc_config = _get_flavor_configuration(
             model_path=model_path, flavor_name=mlflow.pyfunc.FLAVOR_NAME)
+
+    python_model_cloudpickle_version = pyfunc_config.get(CONFIG_KEY_CLOUDPICKLE_VERSION, None)
+    if python_model_cloudpickle_version is None:
```

Comment:
This code is backwards compatible because this code path has not yet been included in an MLflow release. It was introduced in #793. 

What do you mean by "use a base version"?

Reply:
Cool!